README.md

```diff
@@ -106,7 +106,6 @@ Here's what's coming up for Microbundle:
 
 [MIT](https://oss.ninja/mit/developit/)
 
-
-[Rollup]: https://github.com/rollup/rollup
-[Bublé]: https://github.com/Rich-Harris/buble
-[Nodent]: https://github.com/MatAtBread/nodent-compiler
```

Comment:
why, if anything you should correct `Nodent` to `NoDent`

Reply:
I didn't change this line, I only added the line for globals but the precommit hook includes lint-staged which modified the rest of the README

Reply:
Yep, at the beginning, I thought it was due to the prettier extension on my VSCode, and so I've desactivated the format.onSave setting on VSCode

But it behaved the same way when I try to commit after modifying the file with nano...

If you want I can recommit it with --no-verify or without lint-staged

Reply:
just seems really weird that it would lower case that and not the link above

Reply:
🤷‍♂️

Reply:
documentation.js does some strange markdown injection stuff. as long as it still works it's probably safe to ignore haha